src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -266,9 +273,16 @@ std::vector<ov::Tensor> SubgraphBaseTest::calculate_refs() {
     }
 
     const auto& outputs = functionToProcess->outputs();
+    size_t gapSize = outputs.size() - outType.size();
```

Comment:
the same as previous comment

Reply:
Fixed in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/c947eb2125e38214427758b19a94264a952d5856.

Reply:
Please revove the code under comment